Manuscript file upload progress "stuck" at 0%

**Describe the bug**
When uploading a manuscript file on reviewer.elifesciences.org it shows at 0% file upload progress (i.e. doesn't count up). After refreshing a few times it shows as uploaded so assume client side is not showing progress despite upload "working".

**Steps To Reproduce**
Upload a manuscript file!

**Expected behavior**
Client will count up to 100% to feedback upload

**Device, Operating System and Browser**
Chrome 75 on Mac OSX Mojave

Running `npm ls graphql` gives the following clues:

```
├─┬ pubsweet-client@8.0.0
│ ├─┬ apollo-link@1.2.2
│ │ └── graphql@14.4.2  extraneous
│ ├─┬ apollo-link-context@1.0.8
│ │ └─┬ apollo-link@1.2.3
│ │   └── graphql@14.0.2  extraneous
│ └─┬ apollo-link-http@1.5.4
│   ├─┬ apollo-link@1.2.3
│   │ └─┬ apollo-utilities@1.3.2
│   │   └── graphql@14.4.2  extraneous
│   └─┬ apollo-link-http-common@0.2.4
│     └─┬ apollo-link@1.2.3
│       └─┬ apollo-utilities@1.3.2
│         └── graphql@14.4.2  extraneous
└─┬ pubsweet-server@13.5.0
  ├─┬ apollo-server-express@2.6.4
  │ └─┬ apollo-server-core@2.6.4
  │   └─┬ apollo-engine-reporting@1.3.2
  │     ├─┬ apollo-graphql@0.3.2
  │     │ └── graphql@14.3.1  extraneous
  │     ├─┬ apollo-server-core@2.6.4
  │     │ ├── graphql@14.3.1  extraneous
  │     │ ├─┬ graphql-subscriptions@1.1.0
  │     │ │ └── graphql@14.3.1  extraneous
  │     │ └─┬ subscriptions-transport-ws@0.9.16
  │     │   └── graphql@14.0.2  extraneous
  │     └── graphql@14.3.1  extraneous
  ├── graphql@14.3.1 
  └─┬ subscriptions-transport-ws@0.9.16
    └── graphql@14.0.2  extraneous

npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/apollo-link/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link-http-common/node_modules/apollo-utilities/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link-http/node_modules/apollo-utilities/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-engine-reporting/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-graphql/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-server-core/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/graphql-subscriptions/node_modules/graphql
npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/subscriptions-transport-ws/node_modules/graphql
npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-server/node_modules/subscriptions-transport-ws/node_modules/graphql
```

One solution is to use resolutions:
https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

--- a/package.json
+++ b/package.json
@@ -93,8 +93,8 @@
     "secretcheck"
   ],
   "resolutions": {
-    "lodash": ">=4.17.11",
-    "pubsweet-client/graphql": "^14.2.1"
+    "graphql": "^14.2.1",
+    "lodash": ">=4.17.11"
   },
   "workspaces": [
     "client/*",

--- a/package.json
+++ b/package.json
@@ -93,7 +93,8 @@
     "secretcheck"
   ],
   "resolutions": {
-    "lodash": ">=4.17.11"
+    "lodash": ">=4.17.11",
+    "pubsweet-client/graphql": "^14.2.1"
   },
   "workspaces": [
     "client/*",
@@ -113,6 +114,7 @@
     "config": "^3.0.1",
     "date-fns": "^1.29.0",
     "formik": "1.5.7",
+    "graphql": "^14.2.1",
     "graphql-tag": "^2.8.0",
     "history": "^4.7.2",
     "html-pdf": "^2.2.0",

